source/trx2junit/Internal/junit2trx/TrxTestResultXmlBuilder.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System;
+using System.Xml.Linq;
+using trx2junit.Models;
+
+namespace trx2junit
+{
+    public class TrxTestResultXmlBuilder : TrxBase, ITestResultXmlBuilder<TrxTest>
+    {
+        private static readonly Guid s_testTypeId = Guid.Parse("13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b");
```

Comment:
Visual Studio can open trx-files, but not individual tests. So this exact guid has to be used, in order to open the tests.
Found this guid by looking at other _trx_-files.

Maybe there are some other distinct guid-values. I don't know. If there is any special value, please let me know.